resources/keywords/create_instance.resource

```diff
@@ -0,0 +1,90 @@
+*** Settings ***
+Resource    ../keywords/deploy_instance_dev.resource
+Resource    ../object_repo/create_dbinstance_obj.resource
+Resource    ../object_repo/database_access_obj.resource
+Library     ../../utils/scripts/util.py
+
+*** Variables ***
+${instanceName}      ${EMPTY}
+
+*** Keywords ***
+User Navigates To Create Database Instance Screen From Database Access Page
+    User Navigates To Database Access Under Data Services
+    Wait Until Page Contains Element    ${titleDatabaseAccess}
+    Select Database Instance From Create Button DropDown
+
+User Selects DBProvider And Invalid Provider Account
+    [Arguments]    ${isv_name}=${isv}
+    Wait Until Element Is Visible    ${drpDwnDBProvider}    timeout=20s
+    Sleep    3s
+    Select From List By Label    ${drpDwnDBProvider}    ${isv_name}
+    Wait Until Page Contains Element    ${drpDwnDevCatProviderAccount}     timeout=10s
+    Select From List By Label    ${drpDwnDevCatProviderAccount}     ${provaccname_inv}
+
+User Selects DBProvider And Provider Account
+    [Arguments]    ${isv_name}=${isv}
+    Wait Until Element Is Visible    ${drpDwnDBProvider}    timeout=20s
+    Sleep    3s
+    Select From List By Label    ${drpDwnDBProvider}    ${isv_name}
+    Wait Until Page Contains Element    ${drpDwnDevCatProviderAccount}     timeout=10s
+    Select From List By Label    ${drpDwnDevCatProviderAccount}     ${provaccname}
+
+User Enters Data To Create CrunchyDB Database Instance On Admin View
+    User Selects DBProvider And Provider Account
+    ${instanceName}=    Get Instance Name   adm
+    Set Test Variable    ${instanceName}
+    Input Text    ${txtBxInstName}    ${instanceName}
+    Element Should Be Visible    ${btnCreateEnabled}
+    Click Element    ${btnCreateEnabled}
+
+User Enters Data To Create CrunchyDB Database Instance On Dev
```

Comment:
Is this keyword to perform creating instance from Developers view? If yes, from line 42 to 45 we cannot select provider account and database provider, all these would be auto populated